package.json

```diff
@@ -33,17 +34,23 @@
     "babel-loader": "^8.0.6",
     "bcrypt": "^3.0.8",
     "body-parser": "^1.19.0",
+    "cookie-session": "^1.4.0",
     "dotenv": "^6.2.0",
     "express": "^4.17.1",
     "express-validator": "^6.4.0",
     "i18n": "^0.8.5",
     "jsonwebtoken": "^8.5.1",
     "localStorage": "^1.0.4",
+    "node-mocks-http": "^1.8.1",
```

Comment:
is this used somewhere? let's not include unused packages,

Reply:
I've removed it.